american-truck-simulatorconfig.json

```diff
@@ -0,0 +1,326 @@
+    },
+    {
+        "DisplayName":"Max AI Vehicles Player Spawn",
+        "Category":"ATS - Server Settings",
+        "Description":"",
+        "Keywords":"max,ai,vehicles,player,spawn",
+        "FieldName":"max_ai_vehicles_player_spawn",
+        "InputType":"number",
+        "IsFlagArgument":false,
+        "ParamFieldName":"max_ai_vehicles_player_spawn",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"50",
+        "MinValue":"0",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"connection_dedicated_port",
+        "Category":"ATS - Server Settings",
+        "Description":"Port used for main game traffic",
+        "Keywords":"",
+        "FieldName":"$MainServerPort",
+        "InputType":"hidden",
+        "IsFlagArgument":false,
+        "ParamFieldName":"connection_dedicated_port",
+        "IncludeInCommandLine":false,
+        "EnumValues":{},
+        "Hidden":true
+    },
+    {
+        "DisplayName":"query_dedicated_port",
+        "Category":"ATS - Server Settings",
+        "Description":"Port used for Steam Query",
+        "Keywords":"",
+        "FieldName":"$QueryDedicatedPort",
+        "InputType":"number",
+        "IsFlagArgument":false,
+        "ParamFieldName":"query_dedicated_port",
+        "IncludeInCommandLine":false,
+        "EnumValues":{},
+        "Hidden":true
+    },
+    {
+        "DisplayName":"Server Logon Token",
+        "Category":"ATS - Server Settings",
+        "Description":"Use appid 270880 to create a [Steam Server Login Token](https://steamcommunity.com/dev/managegameservers)",
+        "Keywords":"server,logon,token",
+        "FieldName":"server_logon_token",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"server_logon_token",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"",
+        "Placeholder":"6544F7E034119F113526E96474F",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Player Damage",
+        "Category":"ATS - Server Settings",
+        "Description":"Flag if player can receive damage from other players.",
+        "Keywords":"player,damage",
+        "FieldName":"player_damage",
+        "InputType":"checkbox",
+        "IsFlagArgument":false,
+        "ParamFieldName":"player_damage",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"true",
+        "EnumValues":{
+            "False": "false",
+            "True": "true"
+        }
+    },
+    {
+        "DisplayName":"Traffic",
+        "Category":"ATS - Server Settings",
+        "Description":"Flag if traffic is enabled.",
+        "Keywords":"traffic",
+        "FieldName":"traffic",
+        "InputType":"checkbox",
+        "IsFlagArgument":false,
+        "ParamFieldName":"traffic",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"true",
+        "EnumValues":{
+            "False": "false",
+            "True": "true"
+        }
+    },
+    {
+        "DisplayName":"Hide in Company",
```

Comment:
All words to be capitalised.

Reply:
I won't get into the discussion of minor words in titles.

Reply:
Narky

Reply:
I've updated all this in my dev branch. But hold until I've fully tested.